python/ruff-ecosystem/ruff_ecosystem/format.py

```diff
@@ -177,3 +234,19 @@ async def ruff_format(
 
     lines = result.decode("utf8").splitlines()
     return lines
+
+
+class FormatComparison(Enum):
+    # Run Ruff baseline then Ruff comparison; checks for changes in behavior when formatting previously "formatted" code
```

Comment:
Python noob question: What's the reason for using inline comments here instead of docstrings?

Reply:
I don't think there's a formalized way to document class attributes individually. Generally what you do is create a class docstring and then document the members using an 'Attributes' section, but that felt like more work than I wanted to do here 😬 

tldr a docstring for the class is proper but I was lazy. I wish we had tooling to generate the scaffold.

Reply:
I even just wrote it out and it feels awkward; I guess I write Rust now

```
 class FormatComparison(Enum):
+    """
+    The type of format comparison to use.
+
+    Attributes:
+        ruff_then_ruff: Run Ruff baseline then Ruff comparison; checks for changes in
+            behavior when formatting previously "formatted" code
+        ruff_and_ruff: Run Ruff baseline then reset and run Ruff comparison; checks
+            changes in behavior when formatting "unformatted" code
+        black_then_ruff: Run Black baseline then Ruff comparison; checks for changes in
+            behavior when formatting previously "formatted" code
+        black_and_ruff: Run Black baseline then reset and run Ruff comparison; checks
+            changes in behavior when formatting "unformatted" code
+    """
```

Reply:
https://peps.python.org/pep-0258/#attribute-docstrings
https://github.com/microsoft/pylance-release/issues/1576

Reply:
Since you nudged, I switched to individual attribute docstrings — while the PEP is rejected, VSCode supports it.